docs/content/tutorials/index.md

```diff
@@ -116,21 +117,13 @@ All persistent state such as the cluster metadata store and segments for the ser
 
 Later on, if you'd like to stop the services, CTRL-C to exit the `bin/start-micro-quickstart` script, which will terminate the Druid processes.
 
-### Resetting cluster state
-
-If you want a clean start after stopping the services, delete the `var` directory and run the `bin/start-micro-quickstart` script again.
-
-Once every service has started, you are now ready to load data.
+Once the cluster has started you can navigate to [http://localhost:8888](http://localhost:8888).
+This is the router node that also serves the Druid console.
 
-#### Resetting Kafka
+![Druid console](../tutorials/img/tutorial-quickstart-01.png "Druid console")
 
-If you completed [Tutorial: Loading stream data from Kafka](./tutorial-kafka.html) and wish to reset the cluster state, you should additionally clear out any Kafka state.
+It takes a few seconds for all the components to fully initialize and you might see some errors if you open the console immediately after starting the services.
```

Comment:
It takes a few seconds for all the Druid processes to fully start up. If you open the console immediately after starting the services, you may see some errors that you can safely ignore.